frontend/hub/collections/CollectionPage/CollectionContents.tsx

```diff
@@ -1,7 +1,226 @@
 import { useTranslation } from 'react-i18next';
```

Comment:
Hey,

Good job on this implementation so far.

Just a couple of quick questions:

1. Is there an existing ticket related to this change?
2. Could we add end-to-end tests for this implementation?

Reply:
Hello, I added link to the issue. We are separating tests for the issues, distributions issue was for example closed without tests. There should be tests issue for this issue as well (I dont see it), but others collections issues have this. So for tabs, I should create issues as well.